Dockerfile.kubearmor

```diff
@@ -0,0 +1,31 @@
+### Builder
```

Comment:
There already exists this file in the repo in `KubeArmor/build` folder. Better to create a symbolic link or best would be to directly use that file instead of creating new one.

Reply:
We cannot use symbolic link, Reason when building Dockerfiles ( it need to copy multiple folders from parent directory ) but it would try to copy from KubeArmor/build/ only if we use sym link. Also we cannot use that directly still it fails to copy folder which is in parent directory.

Reply:
Possibility: We can remove Dockerfile.kubearmor from KubeArmor/build and place it in parent directory instead.